### PR TITLE
Add `here` to installs

### LINKED ISSUE #102: Installation of R packages needs to include `here`
We tell folks to install some R packages ahead of time in the following spots:

* https://github.com/AlexsLemonade/reproducible-research/blob/c206866f0789d8b1e7e93292b45342ead539545b/docs/setup_instructions/mac_installation_instructions.md#installing-necessary-r-packages
* https://github.com/AlexsLemonade/reproducible-research/blob/c206866f0789d8b1e7e93292b45342ead539545b/docs/setup_instructions/windows_installation_instructions.md#r-packages-for-windows
* https://github.com/AlexsLemonade/reproducible-research/blob/c206866f0789d8b1e7e93292b45342ead539545b/docs/setup_instructions/windows_installation_instructions.md#r-packages-for-wsl

`here` should also be installed/listed, because we use it in the R Notebook: https://github.com/AlexsLemonade/rrp-workshop-exercises/blob/575fcb9b8de381da41c63b870556c9aa40c64f20/analyses/mutation_counts/02_mutation-count-plots.Rmd#L73

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #102 

This PR adds `here` to the list of package installs. I also cleaned up the ordering of the vector of packages to all be in the same order, and match order of bullets.

### CODE REVIEW

**Review by jashapiro — APPROVED** [review]
This looks fine, just a disagreement about ordering. I won't be too offended if you keep it as is.

File: docs/setup_instructions/mac_installation_instructions.md [hunk 1]
```diff
@@ -152,7 +153,7 @@ If you need to install any of the required packages, use the function `install.p
 install.packages("optparse")
 
 # Install all necessary packages at once:
-install.packages(c("optparse", "renv", "rmarkdown", "tidyverse"))
+install.packages(c("tidyverse", "rmarkdown", "optparse", "renv", "here"))
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply docs/setup_instructions/mac_installation_instructions.md:156]: I'll reorder the instructions to match this.

File: docs/setup_instructions/mac_installation_instructions.md [hunk 2]
```diff
@@ -126,18 +126,19 @@ Once the installation process is completed, you can now safely delete (eject) th
...
 ### Installing necessary R packages
 
-For this workshop, you will need to have the following R packages installed: 
+For this workshop, you will need to have the following R packages installed:
 
 * `tidyverse`
 * `rmarkdown`
 * `optparse`
 * `renv`
+* `here`
```
Comment (allyhawkins) [inline docs/setup_instructions/mac_installation_instructions.md:135]: You might want to match the list of packages here to the order that they are installed below.

File: docs/setup_instructions/windows_installation_instructions.md [hunk 3]
```diff
@@ -180,7 +181,7 @@ If you need to install any of the required packages, use the function `install.p
 install.packages("optparse")
 
 # Install all necessary packages at once:
-install.packages(c("optparse", "renv", "rmarkdown", "tidyverse"))
+install.packages(c("tidyverse", "rmarkdown", "optparse", "renv", "here"))
```
Comment (allyhawkins) [inline docs/setup_instructions/windows_installation_instructions.md:184]: I would also make these and the packages on line 259 alphabetical if you do that with the ones above.

**Review by allyhawkins — APPROVED** [review]
This looks fine to me, but I would agree with Josh about making things alphabetical throughout and also listing the packages in markdown in the same order that you are installing.

**Pull request merged**